src/main/java/com/sprint/findex_team6/repository/IndexValRepository.java

```diff
@@ -31,8 +31,9 @@ public interface IndexValRepository extends JpaRepository<IndexVal, Long> {
 
   List<IndexVal> findByIndexIdAndBaseDateBetween(Long indexId, LocalDate startDate, LocalDate endDate, Sort sort);
 
-  Optional<IndexVal> findByIndex_IdAndBaseDate(Long index_id, LocalDate baseDate);
-  
+  List<IndexVal> findByIndexIdAndBaseDateBetween(LocalDate startDate, LocalDate endDate, Sort sort);
+  Optional<IndexVal> findAllByIndex_IdAndBaseDate(Long indexId, LocalDate baseDate);
+
```

Comment:
```suggestion
  Optional<IndexVal> findByIndex_IdAndBaseDate(Long indexId, LocalDate baseDate);
```
Index_val 테이블에는 index_id와 base_date가 unique 제약 조건이 걸려 있습니다. 이렇게 수정하는 것이 좋아 보이네요.

Reply:
네 수정했습니다!!